libs/atqr/domain/src/lib/notifications/notification.valueObject.ts

```diff
@@ -0,0 +1,109 @@
+import { ValueObject } from '../../utils/valueObject';
+import { Guid } from '@tokilabs/lang';
+import { NotificationLogEntry } from './notificationLogEntry.valueObject';
+import { isBeforeDay } from '../../utils/isBeforeDay';
+
+enum NotificationCategory {
```

Comment:
Enums implementation from Notifications.disl are not a part of your task. It's an issue assigned to another person.